src/main/java/racingcar/domain/Race.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.domain;
+
+import java.util.List;
+
+import racingcar.config.RaceConfig;
+import racingcar.dto.CarMovementResponse;
+import racingcar.dto.WinnerResponse;
+
+public class Race {
+	private final List<Car> cars;
+
+	public Race(List<Car> cars) {
+		this.cars = cars;
+	}
+
+	public void moveEachCars(int randomNumber) {
+		cars.forEach(car -> car.move(randomNumber));
+	}
+
+	public List<CarMovementResponse> findRaceResult() {
+		return cars.stream()
+			.map(CarMovementResponse::new)
+			.toList();
+	}
+
+	public List<WinnerResponse> selectWinners() {
+		int maxDistance = findMaxDistance();
+
+		return cars.stream()
+			.filter(car -> car.getDistance() == maxDistance)
```

Comment:
```suggestion
			.filter(car -> car.isSameDistance(maxDistance))
```
Car 클래스 안에 isSameDistance() 같이 메서드로 묶어서 표현하면
읽기좋은 코드가 될것같습니다.

Reply:
`==` 정도는 메서드로 뺄만큼 복잡하지 않다고 생각해서 그대로 썼습니다! 조건문이 더 복잡해진다면 말씀하신 것처러 메서드로 분리해야 할 것 같아요 😀